### ISSUE TITLE
make install fails: module.pod - Non-ASCII character seen

### ISSUE THREAD

**Issue by mathsen** [TACC/Lmod#714]
Since the update to 8.7.43 I'm not able to compile lmod any longer (even now with 8.7.44).
During "make install" I get:

```
/usr/bin/core_perl/pod2man ~/Downloads/lmod-install/lmod/8.7.44/share/man/man1/module.pod   > ~/Downloads/lmod-install/lmod/8.7.44/share/man/man1/module.1
~/Downloads/lmod-install/lmod/8.7.44/share/man/man1/module.pod around line 57: Non-ASCII character seen before =encoding in 'fr'. Assuming UTF-8
POD document had syntax errors at /usr/bin/core_perl/pod2man line 69.
make: *** [makefile:266: man_pages] Error 255
```

manually calling
`/usr/bin/core_perl/pod2man ~/Downloads/lmod-install/lmod/8.7.44/share/man/man1/module.pod   > ~/Downloads/lmod-install/lmod/8.7.44/share/man/man1/module.1`

returns the same error message. Adding
`=encoding UTF-8`
at the beginning of the file fixes the issue for me.

**Comment by rtmclay** [comment]
I have modified the module.pod generation to add **=encoding UTF-8" at the top of this file.  Please test Lmod's "IS714-utf8" branch, to see if it works for you.  Thanks for the bug report!

**Comment by mathsen** [comment]
@rtmclay Thanks for the quick fix! I can confirm that the "IS714-utf8" branch compiles just fine for me.

**Comment by rtmclay** [comment]
This fix is now live in Lmod 8.7.45.  Closing this issue.
